### PR TITLE
fix warning logic

### PR DESCRIPTION
**Pull request by rtesra**
This PR suppresses the warning raised when users do not select ART attending in model options where no ART data is included.

### CODE REVIEW

**Review by jeffeaton — APPROVED** [review]
Thanks! This looks good. Approving; wait until tests pass to merge.

**Comment by r-ash** [comment]
After this behaviour will be

* No warning if ART data not included
* If art data is included

|artattend|include_art_t1|include_art_t2|result|
| ------------ | ------------------ |-----------------------|-------|
|T|T|T|No warning|
|T|T|F|No warning|
|T|F|T|No warning|
|T|F|F|No warning|
|F|T|T|Warn|
|F|T|F|Warn|
|F|F|T|Warn|
|F|F|F|No warning|

**Pull request merged**
